tests/improver-percentiles-to-probabilities/02-basic.bats

```diff
@@ -38,7 +38,7 @@
   run improver percentiles-to-probabilities \
       "$IMPROVER_ACC_TEST_DIR/percentiles-to-probabilities/snow_level.nc" \
       "$IMPROVER_ACC_TEST_DIR/percentiles-to-probabilities/enukx_orography.nc" \
-      "$TEST_DIR/output.nc"
+      "$TEST_DIR/output.nc" "probability_of_falling_snow_level_asl"
```

Comment:
This cube name is not correct or to the standard that was agreed.  Please could you change this to "probability_of_snow_falling_level_below_ground_level".

Reply:
I have updated the name here and in the corresponding KGO.